backend/protocol_rpc/server.py

```
@@ -191,143 +203,161 @@
-                for restore_transaction in restore_transactions:
-                    try:
-                        if (
-                            accounts_manager.get_account(
-                                restore_transaction["to_address"]
+            else:
+                # Restore the transactions
+                try:
+                    newer_transactions = (
+                        local_transactions_processor.get_newer_transactions(tx2["hash"])
+                    )
+                except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)